data-raw/england-icb-secondary-care.R

```diff
@@ -136,23 +139,42 @@ iapt_icb_aggregated <- england_sicb_iapt |>
 
 iapt_icb <- iapt_icb_aggregated |>
   mutate(date = my(date)) |>
-  filter(date >= max(date) %m-% months(2)) |>
+  # filter(date >= max(date) %m-% months(2)) |>
+  filter(date >= as.Date("2023-01-01") & date <= as.Date("2023-03-31")) |>
   group_by(icb22_code) |>
   summarise(percent = mean(value)) |>
   mutate(percent = percent / 100) |>
   mutate(
-    variable = "Talking therapies: \nfinished a course of \ntreatment in 18 weeks \n(Nov 22 - Jan 23 average)",
+    variable = "Talking therapies: \nfinished a course of \ntreatment in 18 weeks \n(Jan 23 - Mar 23 average)",
     number = NA
   ) |>
   relocate(percent, .after = number)
 
+# ---- A&E attendances over 4 hours ----
+attendances_4_hours <- england_icb_accidents_emergency |>
+  mutate(date = parse_date_time(date, orders = "B Y")) |>
```

Comment:
Nice solution. Here is another approach I have used earlier in the script, in case useful to see different approaches:

```r
mutate(date = my(date))
```